pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java

```diff
@@ -855,14 +855,19 @@ private Map<Integer, SegmentZKMetadata> getLatestSegmentZKMetadataMap(String rea
    * idealState.
    * If so, it should create the new segments in idealState.
    *
+   * If the consuming segment is deleted:
```

Comment:
Mention here that this happens only if the special flag  is set

Reply:
Ack